bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/SpProgramFactory.scala

```diff
@@ -70,13 +70,13 @@ object SpProgramFactory {
     prog.setThesis(isThesis(proposal))
 
     prog.setPIInfo(piInfo(proposal))
-    hostNgoEmail(proposal) foreach { e => prog.setPrimaryContactEmail(e) }
+    hostNgoEmail(proposal) foreach prog.setPrimaryContactEmail
 
     // Note: not a typo -- "contact person" is an email
     gemEmail(proposal) foreach { e => prog.setContactPerson(e) }
 
-    minBand3Time(proposal) foreach { tv => prog.setMinimumTime(tv) }
-    timeAcctAllocation(proposal) foreach { alloc => prog.setTimeAcctAllocation(alloc) }
+    minBand3Time(proposal) foreach prog.setMinimumTime
+    timeAcctAllocation(proposal) foreach prog.setTimeAcctAllocation
```

Comment:
That should always be the case, but I don't think we discussed enforcing it; at least, it was never enforced before.

If we want to enforce it, we could do so by throwing an exception so that the skeleton script fails and reports the failure, I guess? This would also log an exception in the ODB, I think.